README.md

```diff
@@ -257,6 +257,32 @@ export default Ember.Route.extend({
 
 If your errors aren't standard, the helper function for that error type can be used as the base to build your custom detection function.
 
+## Usage with Ember Data
+
+Ember AJAX provides a mixin that can be used in an Ember Data Adapter to avoid the networking code provided by Ember Data and rely on Ember AJAX instead. This serves as a first step toward true integration of Ember AJAX into Ember Data.
+
+To use the mixin, you can include the mixin into an Adapter, like so:
+
+```javascript
+// app/adapters/application.js
+import Ember from 'ember';
+import DS from 'ember-data';
+import AjaxServiceSupport from 'ember-ajax/mixins/ajax-support';
+
+const {
+  inject: { service }
+} = Ember;
+const { JSONAPIAdapter } = DS;
+
+export default JSONAPIAdapter.extend(AjaxServiceSupport, {
+  ajaxService: service('ajax'),
```

Comment:
Yeah, I think that's fine. We want to show people that they can allow `ember-ajax` to take care of everything.